sormas-app/app/src/main/res/layout/row_immunization_pick_or_create_layout.xml

```diff
@@ -0,0 +1,183 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="top"
+            android:textSize="@dimen/row_entry_secondary_text_size"
+            android:textColor="@color/row_entry_secondary_text"
+            app:enumValue="@{data.disease}"
+            app:valueCaption="@{I18nProperties.getPrefixCaption(ImmunizationDto.I18N_PREFIX, ImmunizationDto.DISEASE)}"
+            app:defaultValue='@{@string/value_not_available}' />
+
+        <TextView
+            android:id="@+id/immunization_meansOfImmunization"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="top"
+            android:textSize="@dimen/row_entry_secondary_text_size"
+            android:textColor="@color/row_entry_secondary_text"
+            app:enumValue="@{data.meansOfImmunization}"
+            app:valueCaption="@{I18nProperties.getPrefixCaption(ImmunizationDto.I18N_PREFIX, ImmunizationDto.MEANS_OF_IMMUNIZATION)}"
+            app:defaultValue='@{@string/value_not_available}' />
+
+        <TextView
+            android:id="@+id/immunization_immunizationManagementStatus"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="top"
+            android:textSize="@dimen/row_entry_secondary_text_size"
+            android:textColor="@color/row_entry_secondary_text"
+            app:enumValue="@{data.immunizationManagementStatus}"
+            app:valueCaption="@{I18nProperties.getPrefixCaption(ImmunizationDto.I18N_PREFIX, ImmunizationDto.IMMUNIZATION_MANAGEMENT_STATUS)}"
+            app:defaultValue='@{@string/value_not_available}' />
+
+        <TextView
+            android:id="@+id/immunization_immunizationStatus"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="top"
+            android:textSize="@dimen/row_entry_secondary_text_size"
+            android:textColor="@color/row_entry_secondary_text"
+            app:enumValue="@{data.immunizationStatus}"
+            app:valueCaption="@{I18nProperties.getPrefixCaption(ImmunizationDto.I18N_PREFIX, ImmunizationDto.IMMUNIZATION_STATUS)}"
+            app:defaultValue='@{@string/value_not_available}' />
+
+        <TextView
+            android:id="@+id/immunization_externalId"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="top"
+            android:textSize="@dimen/row_entry_secondary_text_size"
+            android:textColor="@color/row_entry_secondary_text"
+            app:value="@{data.externalId}"
+            app:valueCaption="@{I18nProperties.getPrefixCaption(ImmunizationDto.I18N_PREFIX, ImmunizationDto.EXTERNAL_ID)}"
+            app:defaultValue='@{@string/value_not_available}' />
+
+<!--        <LinearLayout-->
```

Comment:
What's the reason for these fields being commented?

Reply:
was not sure if we need them or not, seems like showing too much information there for a row entry
the mechanism that fetches the similar immunization considers the responsibleRegion for now, i am not sure if this is wanted - to be onest i don't get how immunizations are jurisdiction dependent
basically i forgot the commented code there - so the question would be do we want to show in this row entry also the responsible jurisdiction or not?

Reply:
Ah okay, yeah, makes sense. The jurisdiction fields are there because we somehow have to define which users are allowed to access the immunizations (since persons don't have a mandatory jurisdiction themselves). However, the jurisdictions should not be used in the mechanism that retrieves potential duplicates (only disease and startDate/endDate period are relevant here), so that should be changed. I still think that we don't have to display the information here though because mobile app users won't have access to immunizations from other regions and districts anyway, so it's pretty redundant. You can remove the commented code.